src/SFA.DAS.EAS.Account.Api.UnitTests/Orchestrators/StatisticsOrchestratorTests/WhenIGet.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Threading.Tasks;
```

Comment:
How come we are refactoring code in the old location before moving, rather than writing it better once it's in its new home?

Or was it more straight forward to re-factor here and then move the pieces?